File: Week-02/day-03/printer.py
# - Create a function called `printer`
#   which prints the input parameters
#   (can have multiple number of arguments)

def printer(*args):
    for i in args:
        print(i)

printer(1, "two", 3)
printer("apple")
printer("apple", "pear", "melone", "orange")

File: Week-07/Day-03/printer.js
'use strict';
// - Create a function called `printer`
//   which logs to the console the input parameters
//   (can have multiple number of arguments)

function printer(name) {
  for (var i = 0; i < arguments.length; i++) {
    console.log( arguments[i] );
  }
}


printer("Tojas")
printer("Barbi", "CEO")
